XU-TechnicalPaper.md

```diff
@@ -284,27 +354,29 @@ Flawed incentivisation is believed to be the main reason why other solutions for
 
 **Incentives for Exchanges**
 - As discussed in chapter 3.3, the main use of XUC is to incentivize liquidity providers, which in particular targets 'large' exchanges
-- In order to further increase interest of exchanges joining Exchange Union, especially in the beginning, the idea is to allocate a certain amount of XUC to new exchanges joining Exchange Union. These are to be distributed to an exchange's users ('airdrop'). Depending on the business model of the exchange (exchange pays taker's XUC fee, or user directly), the advantage would be that all users can start using XU right away.
+- In order to further increase interest of exchanges joining Exchange Union, especially in the beginning, the idea is to allocate a certain amount of XUC to new exchanges joining Exchange Union. These are to be distributed to an exchange's users ('airdrop'). Depending on the business model of the exchange (exchange pays taker's XUC fee, or user directly), the advantage would be that all users can start using XU as takers right away.
```

Comment:
I kind of like this better as written before, as it's more flexible. If XUC is to be used for staking purposes, which I think is a good idea, then it might be necessary to have XUC to use exchange union at all.

Reply:
agree, changed back